src/Domain/DomainCreators/Shell.hpp

```diff
@@ -95,7 +103,9 @@ class Shell : public DomainCreator<3, TargetFrame> {
       "the radial gridpoints will be spaced uniformly in \f$log(r)\f$. The\n"
       "user may also choose to use only a single wedge (along the -x\n"
       "direction), or four wedges along the x-y plane using the `WhichWedges`\n"
-      "option."};
+      "option. Using the RadialBlockLayers option, a user may change the\n"
+      "Block to Element ratio of the domain, which is useful for testing\n"
```

Comment:
I would argue this is useful in general, not just testing purposes. Say if you want higher radial than angular resolution, then this option let's you easily do that.